ios/Pangea.xcodeproj/project.pbxproj

```diff
@@ -1577,12 +1576,13 @@
 			);
 			inputPaths = (
 				"${SRCROOT}/Pods/Target Support Files/Pods-Pangea/Pods-Pangea-resources.sh",
-				"$PODS_CONFIGURATION_BUILD_DIR/QBImagePickerController/QBImagePicker.bundle",
+				"${PODS_CONFIGURATION_BUILD_DIR}/QBImagePickerController/QBImagePicker.bundle",
```

Comment:
why did this change? I am a bit paranoid regarding all apple config files.

Reply:
@florianlenz The same line got removed and added and again. Don't know why